providers/implementations/rands/drbg_hmac.c

```diff
@@ -0,0 +1,421 @@
+/*
+ * Copyright 2011-2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * HMAC low level APIs are deprecated for public use, but still ok for internal
```

Comment:
This seems to be using EVP_MAC...so what low level APIs are being called here?

Reply:
None, I converted them to EVP_MAC while moving the code across.
Comment and include gone.